packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx

```diff
@@ -0,0 +1,191 @@
+                          flexDirection: 'row',
+                          flexShrink: 0,
+                          paddingBottom: 5,
+                        }}
+                      >
+                        <View style={{ flex: 1 }}>
+                          <Block
+                            style={{
+                              ...styles.mediumText,
+                              fontWeight: 500,
+                              marginBottom: 5,
+                            }}
+                            role="heading"
+                          >
+                            {report.data.name}
+                          </Block>
+                          <DateRange
+                            start={report.data.startDate}
+                            end={report.data.endDate}
+                          />
+                        </View>
+                      </View>
+
+                      {report.data.data ? (
+                        <ChooseGraph
+                          startDate={report.data.startDate}
+                          endDate={report.data.endDate}
+                          data={report.data.data}
+                          mode={report.data.mode}
+                          graphType={report.data.graphType}
+                          balanceType={report.data.balanceType}
+                          groupBy={report.data.groupBy}
+                          compact={true}
+                          style={{ height: 'auto', flex: 1 }}
+                        />
+                      ) : (
+                        <LoadingIndicator />
+                      )}
+                    </View>
+                  </ReportCard>
+                </View>
+                <View
+                  style={{
+                    textAlign: 'right',
+                    position: 'absolute',
+                    right: 25,
+                    top: 25,
+                  }}
+                >
+                  <MenuButton
+                    onClick={() => onMenuOpen(report.data.id, true)}
+                    style={{
+                      color:
+                        isCardHovered === report.data.id
+                          ? 'inherit'
+                          : 'transparent',
+                    }}
+                  />
+                  {reportMenu[report.data.id] && (
+                    <CardMenu
+                      onMenuSelect={onMenuSelect}
+                      onClose={() => onMenuOpen(report.data.id, false)}
+                      reportId={report.data.id}
+                    />
+                  )}
+                </View>
+              </View>
+            ))}
+            {remainder !== 3 &&
+              i + 1 === groupedData.length &&
+              [...Array(remainder)].map((e, i) => (
+                <View key={i} style={{ flex: 1 }} />
+              ))}
+          </View>
+        ))}
+      </>
+    );
+  };
+
+  return reports.length > 0 && ReportGrid(reports);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you move the JSX down here instead of creating an inline component (there should actually be disallowed by eslint.. I'm surprised you managed to bypass it).

```suggestion
  if (reports.length === 0) return null;
  return (
     <div /> // something
  );
```